packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2239,6 +2241,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
         const result = await this.summarizerNode.refreshLatestSummary(
             undefined,
+            snapshotRefSeq,
```

Comment:
Any reason to call it `snapshotRefSeq` in here instead of `summaryRefSeq`?

Reply:
They are used interchangeably in runtime. Did not want to tweak existing variable :)